vendor/assets/stylesheets/dvl/components/page_header.scss

```diff
@@ -55,6 +58,22 @@
 
   &.with_back_arrow {
     padding-left: $rhythm * 5;
+    h2 {
```

Comment:
Can we un-nest this one level? I don't think it needs the `&.with_back_arrow` specificity, right?